tmk_core/protocol/vusb/usbconfig.h

```diff
@@ -343,10 +351,18 @@ section at the end of this file).
 
 /* Set INT1 for D- falling edge to count SOF */
 /* #define USB_INTR_CFG            EICRA */
+#ifndef USB_INTR_CFG_SET
 #define USB_INTR_CFG_SET        ((1 << ISC11) | (0 << ISC10))
+#endif
 /* #define USB_INTR_CFG_CLR        0 */
 /* #define USB_INTR_ENABLE         EIMSK */
+#ifndef USB_INTR_ENABLE_BIT
 #define USB_INTR_ENABLE_BIT     INT1
+#endif
 /* #define USB_INTR_PENDING        EIFR */
+#ifndef USB_INTR_PENDING_BIT
 #define USB_INTR_PENDING_BIT    INTF1
+#endif
+#ifndef USB_INTR_VECTOR
 #define USB_INTR_VECTOR         INT1_vect
+#endif
```

Comment:
```suggestion
#define USB_INTR_CFG_SET        ((1 << ISC11) | (0 << ISC10))
/* #define USB_INTR_CFG_CLR        0 */
/* #define USB_INTR_ENABLE         EIMSK */
#define USB_INTR_ENABLE_BIT     INT1
/* #define USB_INTR_PENDING        EIFR */
#define USB_INTR_PENDING_BIT    INTF1
#define USB_INTR_VECTOR         INT1_vect
```

Reply:
These *have* to be overridden for attiny85

Reply:
I *think* some or all of this can actually just be commented out, so that V-USB uses its internal default of INT0. I don't see any mention of that interrupt anywhere else but I also don't know where this INT1 stuff came from.

Reply:
Ah, it's because INT1 = D3 = D-.